### PR TITLE
Prep for 0.17.0 release

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@
...
+- Preview support to create, debug, and deploy PowerShell projects
+- Right click in a "function.json" file to add a binding from a template
+- New JavaScript projects will include a package.json, with automatic logic to install any dependencies before debug or deploy
+- Right click on a timer function to execute now
+- If in advanced create mode, you may use a premium plan when creating a Function App in Azure
+- Added templates to the "verified" category, including Event Hub and Durable functions
+
+### Fixed
+
+- [Bugs fixed](https://github.com/Microsoft/vscode-azurefunctions/issues?q=is%3Aissue+milestone%3A%220.17.0%22+is%3Aclosed)
```
Comment (nturinski) [inline CHANGELOG.md]: nit: Link doesn't filter out for only bugs

File: README.md [hunk 2]
```diff
@@ -82,17 +82,17 @@ brew install azure-functions-core-tools
...
+    * Ubuntu 18.10
 
         ```bash
-        wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb
+        wget -q https://packages.microsoft.com/config/ubuntu/18.10/packages-microsoft-prod.deb
         sudo dpkg -i packages-microsoft-prod.deb
         ```
 
-    * Ubuntu 17.10
+    * Ubuntu 18.04
```
Comment (nturinski) [inline README.md]: Is 17.10 no longer supported?

Comment (EricJizbaMSFT) [reply README.md]: It's not worth listing all possible versions of linux. There's plenty not listed here that are still supported. Ultimately I just copy this list from here: https://github.com/Azure/azure-functions-core-tools/blob/dev/README.md

File: CHANGELOG.md (hunk 1 above)
Comment (EricJizbaMSFT) [reply CHANGELOG.md]: :) That was intentional. I've always thought the labeling was pretty arbitrary and preferred to see a list of all issues

**Pull request merged**